compiler/rustc_mir_transform/src/generator.rs

```diff
@@ -195,6 +195,9 @@ const RETURNED: usize = GeneratorSubsts::RETURNED;
 /// Generator has panicked and is poisoned.
 const POISONED: usize = GeneratorSubsts::POISONED;
 
+// The number of reserved variants. See above.
```

Comment:
```suggestion
/// The number of reserved variants. See above.
```

Reply:
Does this comment need to be in the documentation? I thought not.

Reply:
I think it would be good if this was a documentation comment, it's helpful when browsing rustdoc for the compiler. But perhaps the comment should be more detailed, something like the following:

```rust
/// Number of variants to reserve in generator state. Corresponds to 
/// `UNRESUMED` (beginning of a generator) and `RETURNED`/`POISONED` 
/// (end of a generator) states.
```